### DISCUSSION TITLE
How to subscribe to OCA Discussions

### DISCUSSION THREAD

**Discussion by francesco-ooops** [OCA/odoo-community.org#127] [General]
Go to https://github.com/OCA/odoo-community.org

Click Watch button in top right corner > Custom 

![image](https://github.com/OCA/odoo-community.org/assets/55489873/ae3f8bb5-2a30-4549-b8f0-5f0b2f0c05ef)

Check discussions > Clic Apply

![image](https://github.com/OCA/odoo-community.org/assets/55489873/ae73ab99-9bb8-4ffc-9ebd-3cf0d5a7f419)

**Comment by francesco-ooops** [comment]
@yajo can we pin this? I was trying to have someone subscribe the other day and it's not so easy to remember the path

(thanks @Shide for the original instructions)

**Reply by yajo** [reply]
I don't have permissions for pinning discussions. @sbidoul or @simahawk, can you help here?

**Reply by vdewulf** [reply]
I pinned this post on the Discussions panel:
https://github.com/orgs/OCA/discussions
